generate.sh

```diff
@@ -130,21 +131,41 @@ do
     else
         sed -i "s/##Tfm##/${DEFAULT_TFM}/g" "$restoreProjectsDir/$pkgName.$pkgVersion.csproj"
     fi
-done < $INPUT
+done < "$INPUT"
 IFS=$OLDIFS
 
-# Build the projects to generate source and projects
-"$scriptroot/eng/common/build.sh" --build --restore -bl /p:GeneratePackageSource=true /p:GeneratorVersion=2 $@
+if [[ "$generateTextOnly" != "true" ]]; then
+    # Build the projects to generate source and projects
+    "$scriptroot/eng/common/build.sh" --build --restore -bl /p:GeneratePackageSource=true /p:GeneratorVersion=2 -p:RestoreAdditionalProjectSources="\"$feeds\"" "$@"
 
-# Copy source to destination
-pathToGeneratedSrc="$scriptroot/artifacts/generatedSrc"
-pathToRepoSrc="$pathToDestRepo/src/referencePackages/src"
+    # Copy source to destination
+    pathToGeneratedSrc="$scriptroot/artifacts/generatedSrc/"
+    if [ -z "$pathToDestRepo" ]; then
```

Comment:
Perhaps the type option I mentioned earlier would make it easier to share this logic to set the pathToDestRepo which is similar across ref and text-only packages.

Reply:
Yeah, that's true. Will change it together with the `type` option